src/Compilers/VisualBasic/Portable/Scanner/KeywordTable.vb

```diff
@@ -304,9 +304,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax
 
         Private Shared Function EnsureHalfWidth(s As String) As String
             Dim result As Char() = Nothing
-
+            Dim ch As Char
```

Comment:
I don't believe moving the declaration out of the loop changes anything. What am I missing?